### DISCUSSION TITLE
Edit logo for backend from system settings

### DISCUSSION THREAD

**Discussion by mraf** [givanz/Vvveb#143] [Ideas]
Currently left navigation logo and login logo can be changed in admin theme folder or css file 

admin/default/img/logo.png

![obraz](https://github.com/givanz/Vvveb/assets/10253590/2419d6d4-98c0-45b6-b629-5fdc1804abc3)

/admin/default/img/logom.png

![obraz](https://github.com/givanz/Vvveb/assets/10253590/c8ec449d-0e61-488a-9461-ab1c3f7b7e93)

I think far more better will be to have option changing those images from settings or source from site general settings

![obraz](https://github.com/givanz/Vvveb/assets/10253590/bebae40a-b75e-4bc7-8547-1a9d3c00480a)

**Comment by givanz** [comment]
Thanks for suggestion, I changed sidebar logo to show site logo https://github.com/givanz/Vvveb/commit/f9a9d97b4a9c48eb9b29da78e259b8f052e003a1

Login page is currently rendered directly without loading components https://github.com/givanz/Vvveb/blob/master/admin/controller/base.php#L576  and it can't currently  load site component to display current site logo.

The  logo can be changed easily with a plugin until login page rendering is changed to allow component loading.
